addons/repair/models/repair.py

```diff
@@ -358,10 +360,10 @@ def _create_invoices(self, group=False):
                 invoice_vals = current_invoices_list[0]
                 invoice_vals['invoice_origin'] += ', ' + repair.name
                 invoice_vals['repair_ids'].append((4, repair.id))
-                if not invoice_vals['narration']:
+                if  is_html_empty(invoice_vals['narration']):
                     invoice_vals['narration'] = narration
                 else:
-                    invoice_vals['narration'] += '<br/>' + narration
+                    invoice_vals['narration'] += Markup('<br/>') + narration
```

Comment:
Could be great to check that narration is not void either (to do also at line 347 when adding narration in the if).

Reply:
Changes applied.